test/rekt/features/integrationsink/features.go

```diff
@@ -95,3 +100,71 @@ func SuccessTLS() *feature.Feature {
 
 	return f
 }
+
+func OIDC() *feature.Feature {
+	f := feature.NewFeature()
+
+	integrationSink := feature.MakeRandomK8sName("integrationsink")
+	source := feature.MakeRandomK8sName("source")
+	sourceNoAudience := feature.MakeRandomK8sName("source-no-audience")
+
+	//sinkURL := &apis.URL{Scheme: "http", Host: sink}
+
+	event := cetest.FullEvent()
+	event.SetID(uuid.NewString())
+
+	eventNoAudience := cetest.FullEvent()
+	eventNoAudience.SetID(uuid.NewString())
+
+	f.Prerequisite("OIDC authentication is enabled", featureflags.AuthenticationOIDCEnabled())
+	f.Prerequisite("transport encryption is strict", featureflags.TransportEncryptionStrict())
+	f.Prerequisite("should not run when Istio is enabled", featureflags.IstioDisabled())
+
+	f.Setup("install integration sink", integrationsink.Install(integrationSink))
+
+	f.Setup("integrationsink is addressable", integrationsink.IsAddressable(integrationSink))
+	f.Setup("integrationsink is ready", integrationsink.IsReady(integrationSink))
+
+	f.Requirement("install source for ksink", eventshub.Install(source,
+		eventshub.StartSenderToResource(integrationsink.GVR(), integrationSink),
+		eventshub.InputEvent(event),
+		eventshub.AddSequence,
+		eventshub.SendMultipleEvents(2, time.Millisecond)))
+
+	f.Requirement("install source for ksink without audience", func(ctx context.Context, t feature.T) {
+		addr, err := integrationsink.Address(ctx, sourceNoAudience)
+		if err != nil {
+			t.Error(err)
+			return
+		}
+
+		eventshub.Install(sourceNoAudience,
+			eventshub.StartSenderURLTLS(addr.URL.String(), addr.CACerts),
+			eventshub.InputEvent(eventNoAudience))(ctx, t)
+	})
```

Comment:
```suggestion
	f.Requirement("install source for ksink without audience", eventshub.Install(sourceNoAudience,
		eventshub.StartSenderToResourceTLS(integrationsink.GVR(), integrationSink, nil),
		eventshub.InputEvent(eventNoAudience))
	)
```

Reply:
or if the intention is to not pass the audience then, the line `addr, err := integrationsink.Address(ctx, sourceNoAudience)` needs to be `addr, err := integrationsink.Address(ctx, integrationSink)`

Reply:
Also another hint: we have some "oidc conformance" tests, which check if an addressable correctly handles requests, when OIDC is enabled. Check for example [here](https://github.com/knative/eventing/blob/4fd274af684d96912afdb4e86247807398490ee9/test/rekt/broker_test.go#L272-L289), how we did it for the broker